languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/words_infix_s.txt

```diff
@@ -268,4 +268,10 @@ Hibernation
 Imparität
 Kollektion
 Abspaltung
-Benachrichtigung
\ No newline at end of file
+Benachrichtigung
+Freundschaft
+Frieden
+Schwein
```

Comment:
"Schweine..." seems to be more common, according to e.g. Jan Schreiber's list: `grep "Schweine" german.txt` -> 602 matches, vs `grep "Schweins" german.txt` -> 45 matches. With these ambiguous terms, I think we have to leave the word out for now (unless both s and e variants are always okay, which doesn't seem to be the case).

Reply:
Ok I will comment it out in both

Reply:
Actually, I think we should keep only _Schweine_ in `without infix s` (e.g. _Schweineohren_ seems to be correct too...)